packages/api/cms-api/src/generator/generate-crud-input.ts

```diff
@@ -218,7 +218,7 @@ export async function generateCrudInput(
                 decorators.push("@IsString({ each: true })");
             } else if (refType == "integer") {
                 type = "number[]";
-                decorators.push("@Transform(({ value }) => value.map(id) => parseInt(id))");
+                decorators.push("@Transform(({ value }) => value.map((id: string) => parseInt(id)))");
```

Comment:
```suggestion
                decorators.push("@Transform(({ value }) => value.map((id) => parseInt(id)))");
```

Reply:
@johnnyomair I think we need the explicit type, at least I get a lint error if the type is not set.
<img width="825" alt="Screenshot 2023-10-11 at 09 54 14" src="https://github.com/vivid-planet/comet/assets/12497541/49ad51ef-9dbe-4ffe-821b-f013d45923f1">